docs/faq.rst

```diff
@@ -0,0 +1,104 @@
+.. _faq:
+
+Frequently Asked Questions
+==========================
+
+.. topic:: General Fairness Questions
```

Comment:
I'm not sure that I should be using topics like this. Other suggestions welcome

Reply:
Just don't use it? :D Not sure what other suggestions you're looking for